src/main/java/unrc/dose/User.java

```diff
@@ -142,4 +143,35 @@ private void load(final User u, final String name,
         u.set(ADMIN, admin);
     }
 
+	 /**
+     * this method remove logically a user.
+     * @param usernameUser this username is for delete logicaly,
+     * his account associate
+     * @param passwordUser this param is used for confirm the operation
+     * @return a boolean with this exit of remove or no remove the user
+     */
+	public static boolean deleteUser(final String usernameUser,
+		final String passwordUser) {
+		User user = User.findFirst("username = ? and password = ?",
+			usernameUser, passwordUser);
+        if (user != null) {
+			user.set("active_account", false);
+			return true;
+		}
+		return false;
+	}
+
+	/**
+     * this method is for login the user.
+     * @param usernameUser this username is for login
+     * @param passwordUser this param is used for finally operation
+     * @return a boolean for see if coincide username and pass
+     */
+	public static boolean login(final String usernameUser,
```

Comment:
This method doesn't actually log in a user, this method validate a user's credential. The name should state this using something like `validateCredentials`